music21/converter/__init__.py

```diff
@@ -1163,10 +1163,15 @@ def parse(value: Union[bundles.MetadataEntry, bytes, str, pathlib.Path],
     # a midi string, must come before os.path.exists test
     elif not isinstance(value, bytes) and valueStr.startswith('MThd'):
         return parseData(value, number=number, format=m21Format, **keywords)
-    elif not isinstance(value, bytes) and os.path.exists(valueStr):
+    # windows systems can't test existence of filepath names > 260 chars
+    elif (not isinstance(value, bytes)
+          and len(valueStr) <= 260
```

Comment:
this seems too restrictive both for Windows and other os's (why put a windows restriction onto other filesystems).  In windows, valueStr can be > 260 bytes -- up to 32768, but each directory of the valueStr must be smaller than 260 bytes, so "C:\My Windows Hard Drive\Users\Michael Scott Cuthbert's Home Directory\Myke's Documents and Settings\2020 Was a Very Hard Year\Music Files are Here\Keep Looking for Music Files\They are in here somewhere\Keep Digging\Here's my Directory\No not this one\Yes this one\Okay I think this is longer than 260 characters now\File.mxl" should be fully parseable.

We probably need a separate routine: `and osCanLoad(valueStr)` which returns True for non-windows, but checks the path length in Windows to have no parts > 260 and no total length > 32768.

Reply:
Agree that we should not apply this limit to other OSes. Like the idea of a new function.

I think the extended paths in your example require an opt-in admin setting, and only work on  >= Windows 10 v.1607 (2015). Python 3.6 can read this setting but will still raise `ValueError` if it's not set:
https://docs.python.org/3/using/windows.html#removing-the-max-path-limitation

GitHub actions happens to enable this setting on their Windows-latest CI server, but that was confusing some users who missed bugs on ordinary systems that way:
https://github.com/actions/virtual-environments/issues/1484

Is it safer to just try and catch `ValueError`? In your example we will still crash on earlier versions of Windows or where the setting is absent.

Reply:
I only discovered this on GH actions because some of the musicxml strings are > 32768 chars:
https://github.com/jacobtylerwalls/music21/pull/2/checks?check_run_id=1753971325